tensorflow/python/framework/dtypes.py

```diff
@@ -270,6 +293,19 @@ def __hash__(self):
   def size(self):
     return np.dtype(self.as_numpy_dtype).itemsize
 
+# Define data type range of numpy dtype
+dtype_range = {np.bool_: (False, True),
+               np.bool8: (False, True),
+               np.uint8: (0, 255),
+               np.uint16: (0, 65535),
+               np.int8: (-128, 127),
+               np.int16: (-32768, 32767),
+               np.int64: (-2**63, 2**63 - 1),
+               np.uint64: (0, 2**64 - 1),
+               np.int32: (-2**31, 2**31 - 1),
+               np.uint32: (0, 2**32 - 1),
+               np.float32: (-1, 1),
+               np.float64: (-1, 1)}
 
```

Comment:
Define the `dtype_range` for computing the `scale` of dtype, according to [scikit-image dtype range](https://github.com/scikit-image/scikit-image/blob/master/skimage/util/dtype.py#L8)